CHANGELOG.md

```diff
@@ -738,13 +750,15 @@ z.object({})
     1 1 │   <>
     2   │ - → <div·class={`${variable}·px-2·foo·p-4·bar`}/>
       2 │ + → <div·class={`${variable}·foo·bar·p-4·px-2`}/>
-    3 3 │   	<div class={`px-2 foo p-4 bar ${variable}`}/>
+    3 3 │    <div class={`px-2 foo p-4 bar ${variable}`}/>
```

Comment:
Is the changelog formatted by some tools? Most of the new lines are okay, but I'm not sure if we want to change these indentations. Maybe you can just leave the formatting as is and only add the relative changelog entry.

Reply:
Reverted the formatting :)